src/commands/functions/authenticate.js

```diff
@@ -0,0 +1,61 @@
+import {
+  validateSaveUdacityAuthToken,
+} from '.';
+
+const axios = require('axios');
+const prompt = require('prompt');
```

Comment:
We're already using `inquirer` library for prompts, please update code to use `inquirer` instead. 

You can look at `promptInputUdacityAuthToken.js`  or `promptOverwriteDir.js` file as an example